core/eolearn/core/eodata_io.py

```diff
@@ -570,8 +593,8 @@ class FeatureIOJson(FeatureIOGZip[T]):
     """FeatureIO object specialized for JSON-like objects."""
 
     @classmethod
-    def get_file_extension(cls) -> str:
-        return MimeType.JSON.extension
+    def _get_uncompressed_file_extension(cls) -> str:
+        return ".json"
```

Comment:
it's pretty simple to adjust, but won't simplify much. They would still need to be subclasses of `FeatureIOGZip` to ensure backwards compatibility. So it essentially just means that we will do `compress_level=0` instead of `compress_level=compress_level`

If you think this is relevant enough, you can open up an issue, where we discuss this.

Reply:
Also, let's not forget `(g)zless` and `(g)zcat` on *unix systems can read zipped "text" file formats, and one can then simply `gzcat timestamps.json.gz | fx ` to read jsons nicely...